package.json

```diff
@@ -47,7 +47,7 @@
     "ember-cli-template-lint": "^1.0.0-beta.1",
     "ember-cli-uglify": "^2.1.0",
     "ember-data": "~3.5.0",
-    "ember-dictionary": "0.1.0",
+    "ember-dictionary": "0.1.3",
```

Comment:
Should we add a ~ or ^ here?

Reply:
missed that.  ^ will work as it just's list of words.  No breaking code... I will update pr.